.github/workflows/CI.yml

```diff
@@ -96,3 +95,29 @@ jobs:
         with:
           command: test
           args: --benches ${{ matrix.features }}
+
+      - name: Build docs
+        if: matrix.build-only != true
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
```

Comment:
```suggestion
          command: doc
```

Reply:
Oops, I decided to just move it to a separate job completely, no need to build docs on all targets.